php-nginx/README.md

```diff
@@ -19,6 +19,15 @@ docker-compose build php56_nginx
 docker-compose push php56_nginx
 ```
 
+## About
+
+This is a Docker image for PHP (using the php-fpm SAPI) and Nginx. It uses, by
+default, our recommended default configuration, including:
+
+* A default HTTPS only website, with HTTP redirecting to HTTPS
```

Comment:
Agree with you, we should add a bold recommendation but not force that by default, as it wouldn't help newcomers... :/